github-runner-manager/src/github_runner_manager/manager/runner_manager.py

```diff
@@ -183,16 +193,9 @@ def get_runners(self) -> tuple[RunnerInstance, ...]:
         for cloud_runner in cloud_runners:
             if cloud_runner.instance_id not in health_runners_map:
                 runner_instance = RunnerInstance.from_cloud_and_platform_health(cloud_runner, None)
-                runner_instance.health = HealthState.UNKNOWN
                 runner_instances.append(runner_instance)
                 continue
             health_runner = health_runners_map[cloud_runner.instance_id]
-            if health_runner.deletable:
-                cloud_runner.health = HealthState.UNHEALTHY
-            elif health_runner.online:
-                cloud_runner.health = HealthState.HEALTHY
-            else:
-                cloud_runner.health = HealthState.UNHEALTHY
```

Comment:
For comparison:
```
    @property
    def health(self) -> HealthState:
        """Overall health state of the runner instance."""
        if not self.platform_state:
            return HealthState.UNHEALTHY
        if self.platform_state == (
            PlatformRunnerState.BUSY,
            PlatformRunnerState.IDLE,
        ):
            return HealthState.HEALTHY
        return HealthState.UNKNOWN
```

`deletable` is `True` if Platform doesn't exist for given VM.
`online` is `True` if Platform is `idle` or `busy`
Otherwise the health is unhealthy.